bootstrap/templates/git-lfs

```diff
@@ -0,0 +1,15 @@
+#%Module1.0
```

Comment:
If you are moving the git-lfs template, then you also need to update the documentation. I recommend moving only the miniconda template for now. In the original location doc/modulefile_templates, there are now 5 templates altogether (ecflow, git-lfs, miniconda, openmpi, qt). I would only move the miniconda template and check if there is any documentation that needs to be updated.

Reply:
I had added the ability to download git-lfs as well, but dropped it and then these new modules were added in the meantime. I removed git-lfs from this directory.